# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1751935822 -7200
# Parent  317d0ca9d72d76a2d1790435a45c6b557526fa01
Feed Enhancements: Don't include feed-subscriptions.css in Account Manager.

diff --git a/mail/themes/shared/mail/accountManage.css b/mail/themes/shared/mail/accountManage.css
--- a/mail/themes/shared/mail/accountManage.css
+++ b/mail/themes/shared/mail/accountManage.css
@@ -639,4 +639,31 @@ richlistitem[default="true"],
 
 .chat-current-session > span {
   font-weight: bold;
 }
+
+/* Used for feed accounts (copied from feedSubscribe.css to avoid its
+ * inclusion in the account manager (initially from bug 1263873).
+ */
+#autotagPrefix {
+  max-width: 15em;
+}
+
+.icon-info {
+  margin-inline-start: 6px;
+  vertical-align: middle;
+  cursor: help;
+}
+
+#feedOptionsGridAccount {
+  display: grid;
+  grid-template-columns: auto auto;
+  align-items: start;
+}
+
+#feedOptionsGridAccount details {
+  margin-block-start: 10px;
+}
+
+#feedOptionsGridAccount legend {
+  display: inline;
+}
diff --git a/mailnews/extensions/newsblog/am-newsblog.xhtml b/mailnews/extensions/newsblog/am-newsblog.xhtml
--- a/mailnews/extensions/newsblog/am-newsblog.xhtml
+++ b/mailnews/extensions/newsblog/am-newsblog.xhtml
@@ -26,12 +26,8 @@
       content="default-src chrome:; script-src chrome: 'unsafe-inline'; style-src chrome: 'unsafe-inline'"
     />
     <link rel="localization" href="messenger/accountSettings.ftl" />
     <link rel="stylesheet" href="chrome://messenger/skin/accountManage.css" />
-    <link
-      rel="stylesheet"
-      href="chrome://messenger-newsblog/skin/feed-subscriptions.css"
-    />
     <script
       defer="defer"
       src="chrome://messenger/content/globalOverlay.js"
     ></script>
